content/docs/installation/best-practice.md

```diff
@@ -22,6 +22,109 @@ are designed for backwards compatibility rather than for best practice or maximu
 You may find that the default resources do not comply with the security policy on your Kubernetes cluster
 and in that case you can modify the installation configuration using Helm chart values to override the defaults.
 
+## Isolate cert-manager on dedicated node pools
+
+cert-manager is a cluster scoped operator and you should treat it as part of your platform control plane.
```

Comment:
perhaps?
```suggestion
cert-manager is a cluster scoped operator and you should treat it as part of your platform's control plane.
```

this depends on what you mean. If you mean `Platform Control Plane`, then I'd probably write it as such instead...